ARM-wvd-templates/HCI/CreateHciHostpoolTemplate.json

```diff
@@ -0,0 +1,490 @@
+      },
+      "dynamicMemoryConfig": {
+        "type": "bool",
+        "metadata": {
+          "description": "This parameter is optional and only used if dynamicMemory = true. When using dynamic memory this setting is the buffer of extra memory given to the VM."
+        },
+        "defaultValue": false
+      },
+      "targetMemoryBuffer": {
+        "type": "int",
+        "metadata": {
+          "description": "This parameter is optional and only used if dynamicMemory = true. When using dynamic memory this setting is the buffer of extra memory given to the VM."
+        },
+        "defaultValue": 0
+      },
+      "workSpaceName": {
+        "type": "string",
+        "metadata": {
+          "description": "The name of the workspace to be attach to new Applicaiton Group."
+        },
+        "defaultValue": ""
+      },
+      "workspaceLocation": {
+        "type": "string",
+        "metadata": {
+          "description": "The location of the workspace."
+        },
+        "defaultValue": ""
+      },
+      "workspaceResourceGroup": {
+        "type": "string",
+        "metadata": {
+          "description": "The workspace resource group Name."
+        },
+        "defaultValue": ""
+      },
+      "isNewWorkspace": {
+        "type": "bool",
+        "metadata": {
+          "description": "True if the workspace is new. False if there is no workspace added or adding to an existing workspace."
+        },
+        "defaultValue": false
+      },
+      "allApplicationGroupReferences": {
+        "type": "string",
+        "metadata": {
+          "description": "The existing app groups references of the workspace selected."
+        },
+        "defaultValue": ""
+      },
+      "addToWorkspace": {
+        "type": "bool",
+        "metadata": {
+          "description": "Whether to add applicationGroup to workspace."
+        }
+      },
+      "vmAdministratorAccountUsername": {
+        "type": "string",
+        "metadata": {
+          "description": "A username to be used as the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by administratorAccountUsername and domainAdministratorPassword will be used."
+        },
+        "defaultValue": ""
+      },
+      "vmAdministratorAccountPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The password associated with the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by domainAdministratorUsername and domainAdministratorPassword will be used."
+        },
+        "defaultValue": ""
+      },
+      "oUPath": {
+        "type": "string",
+        "metadata": {
+          "description": "OU Path for the domain join"
```

Comment:
change to-
Enter the name of the organizational unit (OU) you would like the virtual machines to join. If you do not have a specific OU, leave this parameter empty. The OU should look like OU=unit1,DC=contoso,DC=com [Learn more](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names?WT.mc_id=Portal-Microsoft_Azure_WVD)